utils/issues/issuescollection.go

```diff
@@ -0,0 +1,301 @@
+package issues
+
+import (
+	"github.com/jfrog/jfrog-cli-security/utils"
+	"github.com/jfrog/jfrog-cli-security/utils/formats"
+	"github.com/jfrog/jfrog-cli-security/utils/jasutils"
+	"github.com/jfrog/jfrog-cli-security/utils/results"
+	"github.com/jfrog/jfrog-cli-security/utils/severityutils"
+)
+
+// Group issues by scan type
+type ScansIssuesCollection struct {
+	formats.ScanStatus
+	results.ResultContext
+
+	LicensesViolations []formats.LicenseViolationRow
+
+	ScaVulnerabilities []formats.VulnerabilityOrViolationRow
+	ScaViolations      []formats.VulnerabilityOrViolationRow
+
+	IacVulnerabilities []formats.SourceCodeRow
+	IacViolations      []formats.SourceCodeRow
+
+	SecretsVulnerabilities []formats.SourceCodeRow
+	SecretsViolations      []formats.SourceCodeRow
+
+	SastViolations      []formats.SourceCodeRow
+	SastVulnerabilities []formats.SourceCodeRow
+}
+
+// General methods
+
+func (ic *ScansIssuesCollection) Append(issues *ScansIssuesCollection) {
+	if issues == nil {
+		return
+	}
+	// Result context should be the same for all collections
+	ic.ResultContext = issues.ResultContext
```

Comment:
shouldn't we implement an 'append' here like in the rest of the fields instead of just replacing the entire ResultsContext?
In the single usage we have for this func, if we iterate over several projects we will get the resultsContext of the last of them only (see auditPullRequest func)